### ISSUE TITLE
add: tag for extra transactions that would have been a good fit for the block; but we didn't know about 

### ISSUE THREAD

**Issue by 0xB10C** [0xB10C/miningpool-observer#7]

**Comment by joostjager** [comment]
Do you mean transactions that showed up in a mined block that in retrospect would have been included in the template?

**Comment by 0xB10C** [comment]
Yes, for example, these three transactions right after the coinbase transaction in this AntPool block: https://miningpool.observer/template-and-block/000000000000000000006b45497499ba62079e4525a12ceda27f1b39b68697b5

If they would have been in my mempool at the time of building the template, my node would probably have included them. They were good canidates.

![image](https://github.com/0xB10C/miningpool-observer/assets/19157360/f59864b2-3b28-4c61-8577-e96cc491ebbc)

**Comment by joostjager** [comment]
I think including them in the template would be valuable. It shows what the miner could have earned in fees given publicly available transactions and anything relayed to them in private.

But technically, I suppose it is difficult? You are using core's getblocktemplate, so when the block comes in, the template will change immediately and there is no opportunity anymore to see what the mempool would've looked like if the block txes were already known.
